src/reporting/branch.se

```diff
@@ -139,17 +148,15 @@ def isContainerForReportingToken(shadyReportingToken: address):
     legitMarket = shadyMarket
     return(legitMarket.isContainerForReportingToken(shadyReportingToken))
 
-def isParentOf(shadyChild: address):
-    branchId = shadyChild.getParentPayoutDistributionHash()
-    return(self.childBranches[branchId] == shadyChild)
```

Comment:
Hmm, I'm not sure why that's showing as having been deleted.  It currently exists in the bond-payment branch: https://github.com/AugurProject/augur-core/blob/bond-payout/src/reporting/branch.se